style.css

```diff
@@ -0,0 +1,27 @@
+*{
+    margin: 0;
+    padding: 0;
+}
+
+
+table {
+    font-family: arial, sans-serif;
+    border-collapse: collapse;
+    width: 65%;
+  }
+  
+  td, th {
+    border: 1px solid #928291;
+    text-align: left;
+    padding: 8px;
+  }
+  
+  tr:nth-child(even) {
+    background-color: #928291;
+  }
+  .tableDiv{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
```

Comment:
@RiyaBasak26 are you focusing on responsiveness as well ?

Reply:
I used 100vh to keep the table in center